vgridjs/src/timeline_track.tsx

```diff
@@ -50,31 +50,63 @@ function x_to_time(x: number, bounds: TimelineBounds, width: number): number {
 
 // Single row of the timeline corresponding to one interval set
 class TimelineRow extends React.Component<TimelineRowProps, {}> {
+  private canvasRef : React.RefObject<HTMLCanvasElement>;
+
+  constructor(props : TimelineRowProps) {
+    super(props);
+    this.canvasRef = React.createRef();
```

Comment:
Small thing, but I've been trying to semi-consistently use `python_casing` instead of `javascriptCasing`.